examples/fewshot_protonet/README.md

```diff
@@ -0,0 +1,102 @@
+# Prototypical Networks for Few-shot Learning
+
+### 1. Description
+
+This demo implements [Prototypical Networks for Few-shot Learning](https://github.com/jakesnell/prototypical-networks) within the `PyKale` framework. The following instructions are about how to use it.
```

Comment:
"The following instructions are about how to use it." does not say anything and should be removed.

Reply:
Deleted.